bytecomp/bytelink.ml

```diff
@@ -371,37 +371,37 @@ let link_bytecode ?final_name tolink exec_name standalone =
        (* The final STOP instruction *)
        output_byte outchan Opcodes.opSTOP;
        output_byte outchan 0; output_byte outchan 0; output_byte outchan 0;
-       Bytesections.record outchan "CODE";
+       Bytesections.record toc_writer Bytesections.Name.CODE;
```

Comment:
Nit: in fact I think you can write just `CODE` here and in every other callsite, the compiler will disambiguate the constructor.

Reply:
applied